api/domain/common/messages.proto

```diff
@@ -17,16 +17,19 @@ syntax = "proto3";
 // This file follows google's gRPC naming conventions:
 // https://cloud.google.com/apis/design/naming_convention
 
+import "validate/validate.proto";
+
 option go_package = "github.com/finleap-connect/monoskope/pkg/api/domain/common";
 
 package common;
 
 // ServiceInformation are the version and commit hash of a service
 message ServiceInformation {
   // version
+  // TODO: is this just a number?
   string version = 1;
   // name
-  string name = 2;
+  string name = 2 [(validate.rules).string = {pattern: "^[a-zA-Z_]+$", max_bytes: 60}];
```

Comment:
We shouldn't validate the name here. That is the name of a component like "Gateway" or "CommandHandler". Only used by the server to send version information to the client. The server is always right. :-)